azure-pipelines.yml

```diff
@@ -88,10 +87,6 @@ stages:
             Write-Host "Sonar project version is '${sonarProjectVersion}'"
             # Set the variable to it can be used by other tasks
             Write-Host "##vso[task.setvariable variable=SONAR_PROJECT_VERSION]$sonarProjectVersion"
-            $path = Get-ChildItem -Path $env:LOCALAPPDATA\Microsoft\VisualStudio -Filter "16.0_*"
```

Comment:
why is this gone?

Reply:
That part of the code was used to force visual studio 16 preview to be used from the move to .net 5, and it is not needed anymore.